instrumentation/opentelemetry_oban/test/opentelemetry_oban_test.exs

```diff
@@ -175,22 +178,23 @@ defmodule OpentelemetryObanTest do
                     )}
 
     assert %{
-             "messaging.destination": "events",
-             "messaging.destination_kind": :queue,
              "oban.job.attempt": 1,
              "oban.job.inserted_at": _inserted_at,
-             "oban.job.job_id": _job_id,
+             "oban.job.job_id": ^id,
              "oban.job.max_attempts": 1,
              "oban.job.priority": 0,
              "oban.job.scheduled_at": _scheduled_at,
              "oban.job.worker": "TestJobThatReturnsError",
-             "messaging.operation": :process,
-             "messaging.system": :oban
+             "messaging.destination.name": "events",
+             "messaging.operation.type": :process,
+             "messaging.system": :oban,
+             "messaging.message.id": ^id,
+             "messaging.client.id": "TestJobThatReturnsError"
            } = :otel_attributes.map(attributes)
 
     [
       event(
-        name: "exception",
+        name: :exception,
```

Comment:
This was already fixed separately in https://github.com/open-telemetry/opentelemetry-erlang-contrib/pull/558